core/scss/components/sidebar/sidebar.scss

```diff
@@ -6,7 +6,7 @@
   .#{$sidebar-class} {
     $parent: &;
     z-index: $zindex-sidebar;
-    height: 100%;
```

Comment:
Let's discuss, was running into issues using this method. I think we could do min-height: 100vh and height: 100% to cover both uses.

Reply:
switched to min-height and height